pkg/processor/build/runtime/java/docker/Dockerfile.handler-builder

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	 http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk:9-slim
+
+RUN apt-get update
```

Comment:
Please follow Docker best practices and:
1. Chain as many commands as you can to prevent unnecessary layers to be created
2. Use the apt-get pattern that cleans up after itself (see pypy dockerfile)

Same for all Dockerfiles in this PR

Reply:
Done